src/Background/estimators.jl

```diff
@@ -133,3 +133,61 @@ end
 
 MMM() = MMM(3, 2)
 estimate_background(alg::MMM, data; dims = :) = alg.median_factor * median(data, dims = dims) - alg.mean_factor * mean(data, dims = dims)
+
+"""
+    Biweight_location(M=nothing, c=6.0)
+
+Estimate the background using robust statistic biweight location.
```

Comment:
"Estimate the background use the robust biweight location statistic." <- this is a bit more grammatically correct and easier to read.